api/src/main/java/io/opencensus/trace/Tracestate.java

```diff
@@ -94,9 +104,11 @@ public Builder toBuilder() {
     private final Tracestate parent;
     @javax.annotation.Nullable private ArrayList<Entry> entries;
 
+    // Needs to be in this class to avoid initialization deadlock because super class depends on
+    // subclass (the auto-value generate class).
```

Comment:
heh

Reply:
That's what IDEA told me as the error when I tried to do that.